.gitlab-ci.yml

```diff
@@ -5,6 +5,7 @@ bullseye-jdk11:
     - apt-get -y install openjdk-11-jdk-headless gradle
   script:
     - gradle build :bitcoinj-wallettool:installDist --stacktrace
+    - sha256sum core/build/libs/*.jar wallettool/build/install/wallet-tool/bin/*  wallettool/build/install/wallet-tool/lib/*.jar
```

Comment:
These commands could perhaps go to the after_script sections?

Reply:
I'll try that.